acceptancetests/assess_budget.py

```diff
@@ -241,12 +242,28 @@ def parse_args(argv):
     add_basic_testing_arguments(parser)
     return parser.parse_args(argv)
 
+def feed_cookie():
```

Comment:
Heh, cute name, I hate to be a spoilsport but it would be nicer to have a descriptive name for what this function actually does.

populate_controller_cookie or existing_cookie_to_controller_cookie or something boring like that.